capplets/mouse/mate-mouse-properties.ui

```diff
@@ -610,6 +617,153 @@ Author: Wolfgang Ulbrich
+                                  </object>
+                                  <packing>
+                                    <property name="expand">False</property>
+                                    <property name="fill">True</property>
+                                    <property name="position">0</property>
+                                  </packing>
+                                </child>
+                                <child>
+                                  <object class="GtkBox" id="hbox2">
+                                    <property name="visible">True</property>
+                                    <property name="can-focus">False</property>
+                                    <property name="spacing">6</property>
+                                    <child>
+                                      <object class="GtkLabel" id="pointer_size_small_label">
+                                        <property name="visible">True</property>
+                                        <property name="can-focus">False</property>
+                                        <property name="label" translatable="yes" comments="small threshold">Small</property>
+                                        <property name="justify">center</property>
+                                        <property name="xalign">1</property>
+                                        <attributes>
+                                          <attribute name="style" value="italic"/>
+                                          <attribute name="scale" value="0.82999999999999996"/>
+                                        </attributes>
+                                      </object>
+                                      <packing>
+                                        <property name="expand">False</property>
+                                        <property name="fill">False</property>
+                                        <property name="position">0</property>
+                                      </packing>
+                                    </child>
+                                    <child>
+                                      <object class="GtkScale" id="pointer_size_scale">
+                                        <property name="visible">True</property>
+                                        <property name="can-focus">True</property>
+                                        <property name="adjustment">adjustment8</property>
+                                        <property name="digits">0</property>
+                                        <property name="draw-value">False</property>
+                                        <child internal-child="accessible">
+                                          <object class="AtkObject" id="pointer_size_scale-atkobject">
+                                            <property name="AtkObject::accessible-description" translatable="yes">Cursor blinks speed</property>
```

Comment:
that looks off

Reply:
Yes,you're right: this is a leftover after a cut+paste from another widget... that is wrong too ! This comes originally from the keyboard ui.
I'll remove these (from other widgets too).

Reply:
Fix pushed.